mlflow/utils/request_utils.py

```diff
@@ -64,7 +64,9 @@ def augmented_raise_for_status(response):
     except HTTPError as e:
         if response.text:
             raise HTTPError(
-                f"{e}. Response text: {response.text}", request=e.request, response=e.response
+                f"{e}. Response text: {response.text}",
+                request=getattr(e, "request", None),
+                response=getattr(e, "response", None),
```

Comment:
`e` is an instance of `HTTPError`, right? `request` and `response` should always exist.

Reply:
If the HTTPError is raised from here, https://github.com/mlflow/mlflow/pull/12011/files#r1609298288
then request and response is None

Reply:
You are right 👍